docker-compose.yml

```diff
@@ -74,3 +74,9 @@ services:
       image: yandex/clickhouse-server:19
       ports:
         - "8123:8123"
+    ec2_metadata:
+      image: jtblin/aws-mock-metadata
```

Comment:
I will swap this out for our fork of https://github.com/timberio/mock-ec2-metadata/